src/assignment/views/AssignmentOverview.tsx

```diff
@@ -637,7 +642,7 @@ const AssignmentOverview: FunctionComponent<AssignmentOverviewProps> = ({
 									<CheckboxDropdownModal
 										label={t('assignment/views/assignment-overview___klas')}
 										id="Klas"
-										options={getLabelOptions(AssignmentLabelType.CLASS)}
+										options={getLabelOptions('CLASS')}
```

Comment:
Waarom niet meer met een enum? Gaf dat problemen ergens? is analoog met de enums die er al waren, vb AssignmentType

Reply:
ah ik lees nu je comment hierboven, maar leek me analoog dus verwarring minimaal 🤷‍♂️

Reply:
Typescript deed op sommige plaatsen moeilijk over `string` => `AssignmentLabelType` typings, daarmee alles naar strings getrokken